bitnami/redis/values.yaml

```diff
@@ -1219,6 +1237,15 @@ sentinel:
     ## @param sentinel.persistence.sizeLimit Set this to enable a size limit for `emptyDir` volumes.
     ##
     sizeLimit: ""
+  ## persistentVolumeClaimRetentionPolicy
```

Comment:
Please apply the formatting correction to this section as well.

Reply:
Formatting applied.